lua/fold_imports.lua

```diff
@@ -504,6 +504,8 @@ function M.setup(user_config)
   vim.api.nvim_create_user_command("FoldImportsToggle", function()
     config.enabled = not config.enabled
     setup_autocommands()
+    -- Reload current buffer
```

Comment:
```suggestion
    -- Reload current buffer to reset/fold imports on start 
```

Besides that I think we should reload all the visible buffers or just clear them out manually. After further thinking this might lead to the additional issues (e.g. unsaved changes in the buffer w/ conflict to the file system) so we might need to propertly clean them out manually

Reply:
I will look into that. Thanks for the guidance

Reply:
What if we save the contents of the current buffer then reload it when toggled

Reply:
I already fixed it